.github/workflows/build_binaries.yml

```diff
@@ -0,0 +1,139 @@
+        run: |
+          mkdir gnat
+          curl https://master.dl.sourceforge.net/project/gnuada/GNAT_GCC%20Mac%20OS%20X/10.1.0/native/gcc-10.1.0-x86_64-apple-darwin15-bin.tar.bz2 | tar -xvz --strip-components 1 -C gnat
+          cd gnat
+          make ins-all prefix=$HOME/gcc version=10.1.0 built_prefix="/Volumes/Miscellaneous/tmp/opt/gcc-10.1.0" machine=x86_64-apple-darwin15
+
+      - name: Whitakers-Words Checkout
+        uses: actions/checkout@v2
+        with: 
+          path: 'whitakers-words'
+
+        # TODO: Fix tests to work with older version of bash. This is a hack.
+      - name: Whitakers-Words Build and Test
+        run: |
+          export PATH="$HOME/gcc/bin:$PATH"
+          cd whitakers-words
+          make all
+
+          brew install bash
+          sed -i '' 's/\/bin/\/usr\/local\/bin/' test/run-tests.sh
+          make test
+          
+      - name: Binary Organize
+        run: |
+          mkdir words
+          mv whitakers-words/LICENCE.txt   words/
+          mv whitakers-words/bin           words/
+          mv whitakers-words/ADDONS.LAT    words/
+          mv whitakers-words/DICTFILE.GEN  words/
+          mv whitakers-words/EWDSFILE.GEN  words/
+          mv whitakers-words/INDXFILE.GEN  words/
+          mv whitakers-words/INFLECTS.SEC  words/
+          mv whitakers-words/STEMFILE.GEN  words/
+          mv whitakers-words/UNIQUES.LAT   words/
+
+        # TODO: How to incorporate build # or version number into filename?
+        # TODO: How to deal with the lack of signing? Will be a pain for end users.
+      - name: Binary Export
+        uses: actions/upload-artifact@v2
+        with:
+          name: words-macOS
+          path: |
+            words
+
+  Linux:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: GNAT Intall
+        run: |
+          sudo apt-get update
+          sudo apt-get install gnat gprbuild
+
+      - name: Whitakers-Words Checkout
+        uses: actions/checkout@v2
+        with: 
+          path: 'whitakers-words'
+
+      - name: Whitakers-Words Build and Test
+        run: |
+          cd whitakers-words
+          make
+          make test
+      
+      - name: Binary Organize
+        run: |
+          mkdir words
+          mv whitakers-words/LICENCE.txt    words/
+          mv whitakers-words/bin            words/
+          mv whitakers-words/ADDONS.LAT     words/
+          mv whitakers-words/DICTFILE.GEN   words/
+          mv whitakers-words/EWDSFILE.GEN   words/
+          mv whitakers-words/INDXFILE.GEN   words/
+          mv whitakers-words/INFLECTS.SEC   words/
+          mv whitakers-words/STEMFILE.GEN   words/
+          mv whitakers-words/UNIQUES.LAT    words/
+
+      - name: Binary Export
+        uses: actions/upload-artifact@v2
+        with:
+          name: words-Linux
+          path: |
+            words
+  
+#   Windows:
+#     runs-on: windows-latest
+
+#     steps:        
+#       - name: GNAT Install
+#         # TODO
```

Comment:
It seems msys2 does have packages, so Windows is hopefully a bit easier to handle than macOS.

https://packages.msys2.org/package/mingw-w64-x86_64-gcc-ada  
https://packages.msys2.org/package/mingw-w64-x86_64-gprbuild

Reply:
Thanks! I will try it that out.

Reply:
So, you would think Windows would be easier. But so far it has been harder.

I've worked out how to set up the gnat and the rest on a local windows machine. However it does not build properly:
`bin/makeinfl INFLECTS.LAT > /dev/null`
`raised LATIN_UTILS.INFLECTIONS_PACKAGE.GIVE_UP : latin_utils-inflections_package.adb:572`
`mingw32-make: *** [Makefile:35: INFLECTS.SEC] Error 1`

It can't make the inflections file for some reason. If I make that file on my mac, copy it over, and then run the last line in the makefile (`bin/makeefil.exe`), then it seems to function.

I installed msys2, updated all of it's packages, and installed the following three packages:
https://packages.msys2.org/package/mingw-w64-x86_64-gcc-ada
https://packages.msys2.org/package/mingw-w64-x86_64-gprbuild
https://packages.msys2.org/package/mingw-w64-x86_64-make

Then using the make (the last link) to build using `make all`

I'm at a bit of a loss as to what to do about it.